views/help/forms/form_bug.hbs

```diff
@@ -2,12 +2,12 @@
 <input type="hidden" name="type" value="problem">
 
 <label class="form-group">
-	<label class="control-label">Kurzbeschreibung des Problems</label>
+	<span class="control-label">Kurzbeschreibung des Problems</span>
 	<input class="form-control" type="text" name="title" placeholder="Kurzbeschreibung des Problems" required>
 </label>
 <label class="form-group">
```

Comment:
that looks a bit odd, why not a div with a label inside instead of a label with a span?